c8/apikeys.py

```diff
@@ -187,10 +211,36 @@ def response_handler(resp):
                 elif resp.body['error'] is True:
                     return False
                 
-        return self._execute(request, response_handler)
+        return self._execute(request, response_handler, customPrefix="/_api")
 
 #------------------------------------------------------------------------
```

Comment:
Again not an issue, but can we remove these kinds of comments as we see them? No need for this brake, I have removed a couple of these as well. Thanks

Reply:
Sure I will do this now

Reply:
Done